plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -1189,16 +1189,16 @@ public void testLikeWithEscapePredicatePushdown()
     }
 
     @Test
-    public void testIsNullPredicatePushdown()
+    public void testIsNullExpressionPredicatePushdown()
     {
         if (!hasBehavior(SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN)) {
-            assertThat(query("SELECT nationkey FROM nation WHERE name IS NULL"))
-                    .isNotFullyPushedDown(FilterNode.class);
```

Comment:
i guess. you need to squash commits

you can improve the commit message to document that this is no longer so dumb a refactor as you wanted it to be